CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [6.0.0] - 2021-19-01
```

Comment:
The day and month are reversed in this date. (We want to use ISO dates, not US ones.)